examples/concepts/javabip/casinoBroken/Constants.java

```diff
@@ -57,7 +56,6 @@ public class Constants {
     public static final String GUESSED = "GUESSED";
     public static final String ENOUGH_FUNDS = "ENOUGH_FUNDS";
     public static final String ALLOWABLE_BET = "ALLOWABLE_BET";
-    public static final String SAFE_GAME_STEP = "SAFE_GAME_STEP";
```

Comment:
Is this deleted because it is unused or because of a technicality?

I wanted to keep the diff between the broken & adjusted version as small as possible, so personally I would like to keep "unused but safe" definitions in there if possible